pkg/arvo/app/chat-hook.hoon

```diff
@@ -3,8 +3,11 @@
 ::  allow sending chat messages to foreign paths based on write perms
 ::
 /-  *permission-store, *chat-hook, *invite-store, *metadata-store,
-    *permission-hook, *group-store, *permission-group-hook  ::TMP  for upgrade
-/+  *chat-json, *chat-eval, default-agent, verb, dbug
+    *permission-hook, *group-store, *permission-group-hook,  ::TMP  for upgrade
+    hook=chat-hook,
+    store=chat-store,
+    view=chat-view
+/+  *chat-eval, default-agent, verb, dbug, store-lib=chat-store
```

Comment:
Wondering if we should just go and move the things in chat-eval into the store lib...

Reply:
done!